apps/i18n/aichat/en_us.json

```diff
@@ -1,12 +1,93 @@
+  "modelCard_publishWarning": "In order to publish, you must fill out a model card",
+  "modelCard_testingHeader": "Testing and Evaluation",
+  "modelCard_testingEditTooltip": "Describe how you tested the chatbot to ensure it was ready for users and would perform as expected.",
+  "modelCard_testingDisplayTooltip": "How the chatbot was tested before being published for general use.",
+  "modelCard_exampleTopicsHeader": "Example Prompts and Topics",
+  "modelCard_exampleTopicsEditTooltip": "Add example prompts the user could consider when using the chatbot. Type in each example prompt into the text box, then click “Add” for each one.",
+  "modelCard_exampleTopicsDisplayTooltip": "Try some of these example prompts to get started.",
+  "modelCardPanelHeader": "Model Card",
+  "modelComparisonCompletionButton": "Finish",
+  "modelCustomizationHeader": "Model Customization",
+  "modelCustomization_comparisonHeader": "Selected model",
+  "modelCustomization_compareButtonText": "Compare Models",
+  "modelCustomization_comparisonTooltipText": "This is the underlying language model being used by the chatbot. Use the dropdown to select from additional fine-tuned models.",
+  "modelCustomization_temperatureTooltipText": "Temperature affects which words are generated as a response. Use the slider to change the temperature.",
+  "modelCustomization_systemPromptTooltipText": "The system prompt controls how the chatbot behaves. Type your instructions into the text box.",
+  "modelCustomization_updateButtonText": "Update",
+  "modelCustomization_sliderIncrease": "Increase",
+  "modelCustomization_sliderDecrease": "Decrease",
+  "modelCustomizationTab_setupText": "Setup",
+  "modelCustomizationTab_retrievalText": "Retrieval",
+  "modelCustomizationTab_modelPublishText": "Publish",
+  "modelCustomizationSaveButtonText": "Save",
+  "modelDescriptionPanel": "Choose a model",
+  "modelResetNotification": "Model customizations and model card information have been reset to default settings.",
+  "modelUpdateText": "{fieldLabel} has been updated to {updatedText}. {timestamp}",
+  "modelUpdateText2": "{fieldLabel} has been updated. {timestamp}",
+  "retrievalCustomizationHeader": "Retrieval",
+  "retrievalCustomizationTooltipText": "Retrieval lets you add new information for a chatbot to reference. Type in each retrieval statement into the text box, then click “Add” for each one.",
+  "saveChangesReminderAlert": "Remember to save your changes",
+  "saveChangesUnsavedAlert": "You have unsaved changes",
+  "saveChangesSucessAlert": "Saved",
   "startOverAichatModelCustomizations": "This will reset this level to its start state and remove any model customizations or model card information you’ve added or changed.",
+  "teacherOnboardingModal_instructionsBullet": "**Instructions**: Clear directions and goals for each level.",
+  "teacherOnboardingModal_modelCustomizationBullet": "**Model Customizations** *(optional)*:  Clear directions and goals for each level.",
```

Comment:
Thanks for adding the bullet! It looks like there's a diff in bulleted text size compared to what is currently used on production. This is probably fine, but maybe do a quick check with Mark (design) that this is okay (maybe in Slack)? Thanks!
<img width="844" alt="before-update" src="https://github.com/user-attachments/assets/3758b4fe-b38b-4df2-b52b-3676c7da10e3" />

<img width="835" alt="after-update" src="https://github.com/user-attachments/assets/60992cda-a2a3-4ec3-9e6e-e1751c482962" />

Reply:
Ran this by Mark and he gave the ok! I'll adjust the description in a sec and we should be good to go. 

![image](https://github.com/user-attachments/assets/44f0875e-b76d-4e49-91a9-af0214b425ab)